src/udev/udev-watch.c

```diff
@@ -191,7 +191,7 @@ static int synthesize_change(Manager *manager, sd_device *dev) {
         _cleanup_(pidref_done) PidRef pidref = PIDREF_NULL;
         r = pidref_safe_fork(
                         "(udev-synth)",
-                        FORK_RESET_SIGNALS|FORK_CLOSE_ALL_FDS|FORK_DEATHSIG_SIGTERM|FORK_LOG|FORK_RLIMIT_NOFILE_SAFE,
+                        FORK_RESET_SIGNALS|FORK_CLOSE_ALL_FDS|FORK_DEATHSIG_SIGTERM|FORK_REOPEN_LOG|FORK_LOG|FORK_RLIMIT_NOFILE_SAFE,
```

Comment:
any chance you can drop FORK_RLIMIT_NOFILE_SAFE while at it? it is only used when we're invoking external binaries in the codebase, and is pointless for a helper process executing our own code.